main.go

```diff
@@ -322,17 +324,17 @@ func (s *Server) readPropertiesFromEnvVars() error {
 
 	// Read Google Analytics parameters
 	if s.GaTrackingID, err = ReadEnvVar("IGN_GA_TRACKING_ID"); err != nil {
-		log.Printf("Missing IGN_GA_TRACKING_ID env variable. GA will not be enabled")
+		log.Print("Missing IGN_GA_TRACKING_ID env variable. GA will not be enabled")
```

Comment:
Consider using `Println`. This applies to other occurrences.

```suggestion
		log.Println("Missing IGN_GA_TRACKING_ID env variable. GA will not be enabled")
```